docs/docs/segment-shell.md

```diff
@@ -18,7 +18,13 @@ Show the current shell name (ZSH, powershell, bash, ...).
   "foreground": "#ffffff",
   "background": "#0077c2",
   "properties": {
-    "prefix": " \uFCB5 "
+    "custom_text": {
```

Comment:
I would rename this to `mapped_shell_names` in line with `mapped_locations`. Other than that, really good work!!!

Reply:
Thanks for the quick response (and the great utility)! Since you're particular about the state of your repo, do you prefer an amended commit or a new one? And if new, what's the appropriate prefix for the commit message?

Reply:
You can create a new one as it will always be a single commit in the end (I can squash this one). Prefix would be `feat` in this case.